plugins/woocommerce-admin/client/products/product-breadcrumbs.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { chevronRightSmall, Icon } from '@wordpress/icons';
+import { Link } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './product-breadcrumbs.scss';
+
+type Breadcrumb = {
+	href?: string;
+	title: string | JSX.Element;
+	type?: 'wp-admin' | 'wc-admin';
+};
+
+export const ProductBreadcrumbs = ( {
+	breadcrumbs,
+}: {
+	breadcrumbs: Breadcrumb[];
+} ) => {
+	const visibleBreadcrumbs =
+		breadcrumbs.length > 3
+			? [
+					breadcrumbs[ 0 ],
+					{
+						title: <>&hellip;</>,
+					},
+					breadcrumbs[ breadcrumbs.length - 1 ],
+			  ]
+			: breadcrumbs;
+
+	return (
+		<span className="woocommerce-product-breadcrumbs">
+			{ visibleBreadcrumbs.map( ( breadcrumb ) => {
+				const { href, title, type } = breadcrumb;
+				return (
+					<>
```

Comment:
```jsx 
<Fragment key={ key }>
```
Let's help React reconciliation in it's work ;)

Reply:
Good call!  Just added in https://github.com/woocommerce/woocommerce/pull/35596/commits/ee4f0e8f97e23064f009f9253e15db447bd87a03